content/docs/details/scaffolder/index.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Scaffolder
+publishedDate: '2022-05-13T10:53:00.0Z'
+description: Initialize projects and trigger automation from within Backstage. You can prompt parameter inputs and combine scaffolder actions in oder to build a scaffolder template to run within Backstage.
+---
+
+# Overview
+Imagine you’re an engineer looking to create a new microservice. You want to get started as quickly as possible, with minimal boilerplate and red-tape to jump through. At the same time, engineering organizations benefit from having consistency in production, and often put gates in place to enforce it.
+
+Instead of creating blockers for engineering teams, Spotify use the scaffolder and quickly create new microservices, while helping to ensure that production remains mostly consistent.
```

Comment:
```suggestion
Instead of creating blockers for engineering teams, you can use the Roadie scaffolder and quickly create new microservices, while helping to ensure that production remains mostly consistent.
```